docs/data/data-grid/components/quick-filter/quick-filter.md

```diff
@@ -0,0 +1,61 @@
+---
+title: React Data Grid - Quick Filter component
+productId: x-data-grid
+components: QuickFilter, QuickFilterControl, QuickFilterClear
+packageName: '@mui/x-data-grid'
+githubLabel: 'component: data grid'
+---
+
+# Data Grid - Quick Filter
+
+<p class="description">A search field that lets users to filter data in the Data Grid.</p>
+
+## Basic usage
+
+The demo below shows the default quick filter configuration.
+
+{{"demo": "GridQuickFilter.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Anatomy
+
+```tsx
+import { QuickFilter, QuickFilterControl, QuickFilterClear } from '@mui/x-data-grid';
+
+<QuickFilter>
+  <QuickFilterControl />
+  <QuickFilterClear />
+</QuickFilter>;
+```
+
+### QuickFilter
+
+`QuickFilter` is the top level component that provides context to child components.
```

Comment:
Same feedback as above on styles - plain text for headers, `<ComponentCase />` in the body.